deployer/deployer-services/src/main/java/io/hyscale/deployer/services/util/K8sResourceDispatcher.java

```diff
@@ -273,15 +299,14 @@ private void deleteResources(String labelSelector, List<CustomResourceKind> appl
         }
         return kindVsObjects;
     }
-    
-    private List<KubernetesResource> getSortedResources(List<Manifest> manifests) throws HyscaleException{
+
+    private List<KubernetesResource> getSortedResources(List<Manifest> manifests) throws HyscaleException {
```

Comment:
I think this is no longer required and can be removed
Moved the sorting logic to applyK8sResources method